File: C/ex05.c
// Calcula salário a partir das horas trabalhadas

#include <stdio.h>

int main() {
    
    float valorHora, horasTrabalhadas, salario;
    
    printf("Informe a quantidade de horas trabalhadas: ");
    scanf("%f", &horasTrabalhadas);
    
    printf("Informe o valor da hora de trabalho: R$");
    scanf("%f", &valorHora);
    
    salario = valorHora * horasTrabalhadas;
    
    printf("Seu salário será R$%.2f", salario);
    
}

File: Python/ex05.py
# Calcula salário a partir das horas trabalhadas

valor_hora = float(input('Valor por hora trabalhada: '))
hora_trabalhada = float(input('Horas trabalhadas no mês: '))
salario = valor_hora * hora_trabalhada

print('Salário: R${:.2f}'.format(salario))